metpy/calc/thermo.py

```diff
@@ -855,3 +855,116 @@ def relative_humidity_from_specific_humidity(specific_humidity, temperature, pre
     return (100 * units.percent *
             mixing_ratio_from_specific_humidity(specific_humidity) /
             saturation_mixing_ratio(pressure, temperature))
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def cape_cin(pressure, temperature, dewpt, parcel_profile):
+    r"""Calculate CAPE and CIN.
+
+    Calculate the convective available potential energy (CAPE) and convective inhibition (CIN)
+    of a given upper air profile and parcel path. CIN is integrated between the surface and
```

Comment:
@jrleeman 's got it right. A parcel that's starting out unstable below the capping inversion still has to get through the cap to reach its LFC, and in most cases it appears our LFC function is correctly placing the LFC at the top of the capping inversion.